artifacts/scripts/util.sh

```diff
@@ -596,6 +596,10 @@ function git-index-clean() {
 }
 
 function fix-godeps() {
+    if [ "${SKIP_GODEPS:-}" = true ]; then
```

Comment:
can we make this more scoped to publisher bot? like: `PUBLISHER_BOT_SKIP_GODEPS`

Reply:
done